tutorials/intro/plot_introduction.py

```diff
@@ -1,404 +1,293 @@
+# with a *rejection dictionary*:
+
+reject_criteria = dict(mag=4e-12,     # 4000 fT
+                       grad=4e-10,    # 4000 fT/cm
+                       eeg=150e-6,    # 150 μV
+                       eog=250e-6)    # 250 μV
+
+###############################################################################
+# We'll also pass the event dictionary as the ``event_id`` parameter (so we can
+# work with easy-to-pool event labels instead of the integer event IDs), and
+# specify ``tmin`` and ``tmax`` (the time relative to each event at which to
+# start and end each epoch). Finally, since we didn't preload the
+# :class:`~mne.io.Raw` data, we'll tell the :class:`~mne.Epochs` constructor to
+# load the epoched data into memory:
+
+epochs = mne.Epochs(raw, events, event_id=event_dict, tmin=-0.2, tmax=0.5,
+                    reject=reject_criteria, preload=True)
+
+###############################################################################
+# Next we'll pool across left/right stimulus presentations so we can compare
+# auditory versus visual responses. To avoid biasing our signals to the
+# left or right, we'll use :meth:`~mne.Epochs.equalize_event_counts` first to
+# randomly sample epochs from each condition to match the number of epochs
+# present in the condition with the fewest good epochs.
+
+conds_we_care_about = ['auditory/left', 'auditory/right',
+                       'visual/left', 'visual/right']
+epochs.equalize_event_counts(conds_we_care_about)  # this operates in-place
+aud_epochs = epochs['auditory']
+vis_epochs = epochs['visual']
+del raw, epochs  # free up memory
+
+###############################################################################
+# Like :class:`~mne.io.Raw` objects, :class:`~mne.Epochs` objects also have a
+# number of built-in plotting methods. One is :meth:`~mne.Epochs.plot_image`,
+# which shows each epoch as one row of an image map, with color representing
+# signal magnitude; the average evoked response and the sensor location are
+# shown below the image:
+
+aud_epochs.plot_image(picks=['MEG 1332', 'EEG 021'])
+
+###############################################################################
+# Estimating evoked responses
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^
 #
-#   1. Extract the max value of each epoch
+# Now that we have our conditions in ``aud_epochs`` and ``vis_epochs``, we can
+# get an estimate of evoked responses to auditory versus visual stimuli by
+# averaging together the epochs in each condition. This is as simple as calling
+# the :meth:`~mne.Epochs.average` method on the :class:`~mne.Epochs` object,
+# and then using a function from the :mod:`mne.viz` module to compare the
+# global field power for each sensor type of the two :class:`~mne.Evoked`
+# objects:
 
-max_in_each_epoch = [e.max() for e in epochs['aud_l']]  # doctest:+ELLIPSIS
-print(max_in_each_epoch[:4])  # doctest:+ELLIPSIS
+aud_evoked = aud_epochs.average()
+vis_evoked = vis_epochs.average()
```

Comment:
Here again, I would show that the shape of evoked.data is n_channels x n_times.

Reply:
I *didn't* add it here, because it opens up a preloading-related can of worms about why evokeds have `data` instead of `get_data()`.